### DISCUSSION TITLE
Using Synology NAS - do not get the config.yml file

### DISCUSSION THREAD

**Discussion by AmauryVanEspen** [TwiN/gatus#851] [Q&A] [answered]
Hi,
using the Synology Container Manager i've deployed your docker image.
I've mapped the file config.yml to be readable, unfortunately, it seems it does not take it 
Could you please tell a bit more about where the config.yml should be ?
Thanks
Amaury

**Comment by TwiN** [comment, marked as answer]
Normally I wouldn't help for platform/hardware-specific issues, but lucky for you, I also have a Synology NAS and one of my many Gatus instances is running on it.

Basically, I just mount a config folder to `/config`, and I create a `config.yaml` file in that folder.

![image](https://github.com/user-attachments/assets/c3503982-1ddb-433c-9671-bd34dd2eb960)
